UnitySDK/Assets/ML-Agents/Scripts/DemonstrationStore.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.IO;
+using UnityEngine;
+
+
+namespace MLAgents
+{
+    /// <summary>
+    /// Demonstration meta-data.
+    /// </summary>
+    [System.Serializable]
+    public struct DemonstrationMetaData
```

Comment:
Curious as to why this is a separate struct as opposed to instance variables for DemonstrationStore.

Reply:
Makes sense - that would be a nice comment for the Demo struct.